### PR TITLE
Documentation for edge probing

### PR DESCRIPTION
**Pull request by iftenney**
Explain how to run things :)

### CODE REVIEW

File: probing/README.md [hunk 1]
```diff
@@ -1,4 +1,81 @@
-# Edge Probing Utils
+# Edge Probing
+
+This is the main page for [What do you learn from context? Probing for sentence structure in contextualized word representations](https://openreview.net/forum?id=SJzSgnRcKX), a.k.a. "Edge Probing"
```
Comment (sleepinyourhat) [inline probing/README.md]: Period?

File: probing/README.md [hunk 2]
```diff
@@ -1,4 +1,81 @@
...
+
+## Getting Started
+
+First, follow the set-up instructions for `jiant`: [Getting Started](../README.md#getting-started). In particular, you'll need to set the following environment variables:
+- `JIANT_PROJECT_PREFIX` to wherever you want experiments to be saved (like 
+  `$HOME/exp`)
+- `JIANT_DATA_DIR` to the directory where you'll download the edge probing data 
+  (like `$HOME/jiant_data`)
```
Comment (sleepinyourhat) [inline probing/README.md]: Suggest that they use the customizable path setup script?

File: probing/README.md [hunk 3]
```diff
@@ -1,4 +1,81 @@
...
+
+## Running the experiments from the paper
+
+We provide a frozen branch, [`edgeprobe_frozen_feb2019`](https://github.com/jsalt18-sentence-repl/jiant/tree/edgeprobe_frozen_feb2019), which should reflect the master branch as of the final version of the paper.
+
+The configs in `jiant/config/edgeprobe_*.conf` are the starting point for the experiments in the paper, but are supplemented by a number of parameter overrides (the `-o` flag to `main.py`). We use a set of bash functions to keep track of these, which are maintained in [`jiant/scripts/edges/exp_fns.sh`](../scripts/edges/exp_fns.sh).
+
+To run a standard experiment, you can do something like:
+```sh
+export NOTIFY_EMAIL="yourname@gmail.com"  # TODO: make it work without this
```
Comment (sleepinyourhat) [inline probing/README.md]: Do we want to leave this here, or make an issue?

File: probing/README.md (hunk 2 above)
Comment (iftenney) [reply probing/README.md]: Fixed the mention on the main page.

File: probing/README.md (hunk 3 above)
Comment (iftenney) [reply probing/README.md]: Fixed it so it's no longer needed.

File: probing/README.md (hunk 1 above)
Comment (iftenney) [reply probing/README.md]: Done.

**Pull request merged**